source/connect/mongoclient.txt

```diff
@@ -0,0 +1,120 @@
+.. _kotlin-sync-mongoclient:
+
+====================
+Create a MongoClient
+====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, client
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+To connect to a MongoDB deployment, you need two things:
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and perform 
+  operations on the MongoDB deployment.
+
+You can also use either of these components to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` object
+to connect to MongoDB.
+
+.. _kotlin-sync-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. TODO, add this as last sentence for ``username:password`` description: For more information about the ``authSource`` 
+.. connection option, see :ref:`kotlin-sync-auth`. 
+
+.. list-table::
```

Comment:
Are these all a direct copy from Java driver?  Do we want to duplicate it like this?  Up to you but seems hard to maintain.

Reply:
Separate sites are maintained for each driver as of right now. For the Kotlin Sync work, we are using a combination of the [PyMongo](https://www.mongodb.com/docs/languages/python/pymongo-driver/current/connect/mongoclient/) and [Kotlin Coroutine](https://www.mongodb.com/docs/drivers/kotlin/coroutine/current/fundamentals/connection/connect/) docs as a model.